src/Altinn.Dan.Plugin.Arbeidstilsynet/Models/Unit.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
```

Comment:
We need to choose; System.Text.Json or Newtonsoft.Json.
If we continue with Newtonsoft, then we should remove unused System.Text-usings.

Reply:
If System.Text.Json covers our needs, why would we add a third-party lib dependency?

Reply:
Newtonsoft is so much better.

Reply:
It has improved quite a bit in .NET5, but Core is still on 3.1 and Common has it as a dependency. We also have a custom attribute we would have to reimplement, so for now we should just use Newtonsoft.

But we should try to get rid of it when getting Core to .NET6.